pkg/manifests/types.go

```diff
@@ -753,6 +753,11 @@ type RemoteWriteSpec struct {
 	QueueConfig *monv1.QueueConfig `json:"queueConfig,omitempty"`
 	// Defines the timeout value for requests to the remote write endpoint.
 	RemoteTimeout string `json:"remoteTimeout,omitempty"`
+	// Enables sending exemplars via remote write. When enabled, Prometheus is
+	// configured to store a maximum of 100,000 exemplars in memory.
+	// Note that this setting only applies to user-defined monitoring. It is not applicable
+	// to default in-cluster monitoring.
+	SendExemplars *bool `json:"sendExemplars,omitempty"`
```

Comment:
I though we could also have a
```
type UWMRemoteWriteSpec struct {
        RemoteWriteSpec
        // Enables sending exemplars via remote write. When enabled, Prometheus is
	// configured to store a maximum of 100,000 exemplars in memory.
	SendExemplars *bool `json:"sendExemplars,omitempty"`
}
```
and then use that in `PrometheusRestrictedConfig`. That would make it clearer what option can be used where. Though not sure how that affects the API doc generation.

Reply:
just to be sure, this is not a blocker for me and we could go ahead as is.